common/unit_test/Test_Traits.cpp

```diff
@@ -18,21 +28,74 @@ using view_types =
                      std::pair<long double, Kokkos::LayoutRight>,
                      std::pair<long double, Kokkos::LayoutStride>>;
 
+// Define all the combinations
+using paired_value_types =
+    tuple_to_types_t<cartesian_product_t<base_real_types, base_real_types>>;
+
+using paired_layout_types =
+    tuple_to_types_t<cartesian_product_t<base_layout_types, base_layout_types>>;
+
+using paired_view_types =
+    tuple_to_types_t<cartesian_product_t<base_real_types, base_layout_types,
+                                         base_real_types, base_layout_types>>;
+
 template <typename T>
 struct RealAndComplexTypes : public ::testing::Test {
   using real_type    = T;
   using complex_type = Kokkos::complex<T>;
+
+  virtual void SetUp() {
+    GTEST_SKIP() << "Skipping all tests for this fixture";
```

Comment:
Do you mean that tests are only compilation and therefore there is no runtime test?
If it is the case, I think using gtest is quite a burden but perhaps the skip explanation can be more straightforward.

Reply:
Yes, this is a compilation test.
I have commented this at the top of this file. 
Maybe better to comment that `gtest` is used for parameterization only?